### ISSUE TITLE
Preprocessing - ModuleNotFoundError: No module named 'udls.generated'

### ISSUE THREAD

**Issue by wwerkk** [acids-ircam/RAVE#200]
Hi, when I run the preprocessing script I am getting the following error:
```
(ravenv2) wwerk@bill:~$ rave  preprocess --input_path ~/training_data/wellPiano/out_48000 --output_path ~/preprocessed_data
Traceback (most recent call last):
  File "/home/wwerk/.conda/envs/ravenv2/bin/rave", line 8, in <module>
    sys.exit(main())
  File "/home/wwerk/.conda/envs/ravenv2/lib/python3.9/site-packages/scripts/main_cli.py", line 34, in main
    from scripts import preprocess
  File "/home/wwerk/.conda/envs/ravenv2/lib/python3.9/site-packages/scripts/preprocess.py", line 17, in <module>
    from udls.generated import AudioExample
ModuleNotFoundError: No module named 'udls.generated'
```

**Comment by caillonantoine** [comment]
Maybe try `pip install --upgrade udls` !

**Comment by wwerkk** [comment]
After removing and setting up a new virtual env I am now getting the error described in https://github.com/acids-ircam/RAVE/issues/160
Would this suggest it's an issue with ffmpeg?

**Comment by XiaowanYi** [comment]
i had the same problem and solved by a simple fix ` pip uninstall udls`  then ` pip install udls`

you can check your udls version by ` pip show udls`
before the fix, mine showed version 1.7 where it is supposed to be 1.0.1 according to https://pypi.org/project/udls/

**Comment by wwerkk** [comment]
solved by reinstalling udls as described above and installing ffmpeg with conda instead of pip
